pymeasure/instruments/jobinyvon/jobinyvon270m.py

```diff
@@ -0,0 +1,360 @@
+        code = self._get_code(ans)
+        if code != 'o':
+            raise IOError(f'Wrong return code from driver, received {code}')
+        return True
+
+    def move_grating_steps(self, nsteps: int):
+        """
+        ABSOLUTE positioning of the grating motor in number of steps.
+        """
+        ans = self.write_read(f'F0,{nsteps - self.gsteps}\r'.encode(), nread=1, timeout=20000)
+        code = self._get_code(ans)
+        if code != 'o':
+            raise IOError(f'Wrong return code from driver, received {code}')
+
+    def get_grating_wavelength(self):
+        """
+        Reading the wavelength from the grating motor of the spectrometer.
+        """
+        return self._lambda_max - ((self._max_steps - self.gsteps) / self._steps_nm)
+
+    def move_grating_wavelength(self, wavelength: float):
+        """
+        ABSOLUTE positioning of the grating motor in wavelength.
+        """
+        steps = int(self._max_steps - int((self._lambda_max - wavelength) * self._steps_nm))
+        self.move_grating_steps(steps)
+
+    def move_entry_slit_steps(self, nsteps: int):
+        """
+        ABSOLUTE positioning of the entry slit motor in number of steps.
+        """
+        ans = self.write_read(f'k0,0,{nsteps - self.entrysteps}\r'.encode(), nread=1, timeout=10000)
+        time.sleep(0.5)
+        code = self._get_code(ans)
+        if code != 'o':
+            raise IOError(f'Wrong return code from driver, received {code}')
+
+    def get_entry_slit_microns(self):
+        """
+        Reading of the ABSOLUTE position of the entrance slit in micrometres.
+        """
+        return self.entrysteps / self._slit_steps_micron
+
+    def move_entry_slit_microns(self, microns: float):
+        """
+        ABSOLUTE positioning of the entry slit in micrometers.
+        """
+        pos_steps = microns * self._slit_steps_micron
+        pos_steps = int(pos_steps)
+        self.move_entry_slit_steps(pos_steps)
+
+    def move_exit_slit_steps(self, nsteps: int):
+        """
+        ABSOLUTE positioning of the exit slit motor in number of steps.
+        """
+        ans = self.write_read(f'k0,2,{nsteps - self.exitsteps}\r'.encode(), nread=1, timeout=10000)
+        time.sleep(0.5)
+        code = self._get_code(ans)
+        if code != 'o':
+            raise IOError(f'Wrong return code from driver, received {code}')
+
+    def get_exit_slit_microns(self):
+        """
+        Reading of the ABSOLUTE position of the exit slit in micrometres.
+        """
+        return self.exitsteps / self._slit_steps_micron
+
+    def move_exit_slit_microns(self, microns: float):
+        """
+        ABSOLUTE positioning of the exit slit in micrometres.
+        """
+        pos_steps = microns * self._slit_steps_micron
+        pos_steps = int(pos_steps)
+        self.move_exit_slit_steps(pos_steps)
+
+    def check_get_errors(self, error):
+        print(error)
+        pass
```

Comment:
What are you doing here?
Residual of testing?

Reply:
Yes, residual testing, we eliminated those lines.

Reply:
That `print` line is still present.
Actually, the `check_get_errors` of the parent class will log errors, which is a lot better than printing, as a logging handler can catch it.

For further reading see the [logging](https://docs.python.org/3/library/logging.html) package of the python standard library (no install required).